pkg/kube/pod.go

```diff
@@ -129,6 +130,11 @@ func CreatePod(ctx context.Context, cli kubernetes.Interface, opts *PodOptions)
 		Spec: patchedSpecs,
 	}
 
+	// Add fixed name to a pod
+	if opts.Name != "" {
```

Comment:
Do we need to not set `GenerateName` if `Name` is set? Or would the `Name` be chosen as precedence always? Should we fail pod creation if both are set?

Reply:
kubernetes give Name as precedence over GenerateName always. Hence it doesnt matter even if the field is set